### PR TITLE
Recipe for emag tech labs rabbitmq bundle

### PR DESCRIPTION
**Pull request by andreea-anamaria**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by andreea-anamaria** [comment]
Linked to https://github.com/eMAGTechLabs/RabbitMqBundle/issues/33

**Comment by danieltoader** [comment]
@Nyholm Sorry to spam you but could you also please help with this pull request? or let me know who would be available to look over it? It's a maintained fork of php-amqplib/RabbitMqBundle. Thanks!

File: emag-tech-labs/rabbitmq-bundle/2.0/config/packages/old_sound_rabbit_mq.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+old_sound_rabbit_mq:
+    connections:
+        default:
+            url: '%env(RABBITMQ_URL)%'
+            vhost: '%env(RABBITMQ_VHOST)%'
+            lazy:     false
```
Comment (Nyholm) [inline emag-tech-labs/rabbitmq-bundle/2.0/config/packages/old_sound_rabbit_mq.yaml:6]: Could we set this to true? 

That is a great performance boost =)

File: emag-tech-labs/rabbitmq-bundle/2.0/config/packages/old_sound_rabbit_mq.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+            read_write_timeout: 3
+
+            # requires php-amqplib v2.4.1+ and PHP5.4+
+            keepalive: false
+
+            # requires php-amqplib v2.4.1+
+            heartbeat: 0
+
+            #requires php_sockets.dll
+            use_socket: true # default false
```
Comment (Nyholm) [inline emag-tech-labs/rabbitmq-bundle/2.0/config/packages/old_sound_rabbit_mq.yaml:17]: `php_sockets.dll`? Isn't that windows only?

Why is this true by default?

**Review by Nyholm — CHANGES_REQUESTED** [review]
Thank you for the ping

File: emag-tech-labs/rabbitmq-bundle/2.0/config/packages/old_sound_rabbit_mq.yaml (hunk 2 above)
Comment (mihaileu) [reply emag-tech-labs/rabbitmq-bundle/2.0/config/packages/old_sound_rabbit_mq.yaml:17]: @andreea-anamaria let's put it to false

Comment (andreea-anamaria) [reply emag-tech-labs/rabbitmq-bundle/2.0/config/packages/old_sound_rabbit_mq.yaml]: Fixed.

File: emag-tech-labs/rabbitmq-bundle/2.0/config/packages/old_sound_rabbit_mq.yaml (hunk 1 above)
Comment (andreea-anamaria) [reply emag-tech-labs/rabbitmq-bundle/2.0/config/packages/old_sound_rabbit_mq.yaml]: Fixed.

**Review by Nyholm — APPROVED** [review]

**Comment by Nyholm** [comment]
Thank you

**Pull request merged**